examples/bpk-component-bottom-sheet/examples.tsx

```diff
@@ -146,7 +146,7 @@ BottomSheetContainerProps,
             Open bottom sheet
           </BpkButtonV2>
           <BpkBottomSheet
-            ariaLabelledby='test'
```

Comment:
Make the default demo more of a valid example - there was no `#test` element. Now screen reader reads the a11y title.